app.js

```diff
@@ -0,0 +1,217 @@
+const bodyParser = require('body-parser')
+const cookieParser = require('cookie-parser')
+const express = require('express')
+const app = express()
+const exphbs = require('express-handlebars')
+const session = require('express-session')
+const expressValidator = require('express-validator')
+const methodOverride = require('method-override')
+const mongoose = require('mongoose')
+const path = require('path')
+const server = require('http').createServer(app)
+
+const io = require('socket.io')(server)
+const flash =require('connect-flash')
+const passport = require('passport')
+const port = process.env.PORT || 3000;
+
+const routes = require('./routes/routes')
+const dbConfig = require('./config/dbConfig')
+const Twit = require('twit')
+const tweet = require('./helpers/twitter')
+const moment = require('moment-timezone')
+const nsLineStations = require('./helpers/nsLineStations')
+const ewLineStations = require('./helpers/ewLineStations')
+const circleLineStations = require('./helpers/circleLineStations')
+const neLineStations = require('./helpers/neLineStations')
+const dtLineStations = require('./helpers/dtLineStations')
+
+
+mongoose.Promise = global.Promise
+mongoose.connect(dbConfig.urllive, {useMongoClient : true})
+.then(()=>{console.log('Mongoose connected')}, (err)=>{console.log(err)})
+
+//read cookies
+app.use(cookieParser());
+// get info from html forms
+app.use(bodyParser.json());
+
+app.use(session({
+  secret: 'project2traindelaytracker',
+  resave: false,
+  saveUninitialized: true,
+}))
+
+// Passport
+
+app.use(passport.initialize());
+app.use(passport.session());
+
+// Flash messages for express
+
+app.use(flash());
+app.use((req, res, next) => {
+  res.locals.alerts = req.flash();
+  res.locals.currentUser = req.user;
+  next();
+});
+
+// Express validator
+
+app.use(expressValidator({
+  errorFormatter : (param, msg, value) => {
+      let namespace = param.split('.'),
+      root = namespace.shift(),
+      formParam = root
+
+        while(namespace.length){
+          formParam += '['+ namespace.shift()+ ']'
+        }
+
+        return {
+          param : formParam,
+          msg : msg,
+          value : value
+        }
+  }
+}))
+
+app.use(bodyParser.urlencoded({extended: true}))
+
+app.use(express.static(path.join(__dirname, 'public')))
+app.engine('handlebars', exphbs({defaultLayout: 'main'}))
+
+app.set('view engine', 'handlebars')
+
+// Routes
+app.use('/', routes)
```

Comment:
Most of your operations happened in sockets, where are the CRUD database operation?